AppCUI/src/Controls/NumericSelector.cpp

```diff
@@ -574,4 +522,79 @@ bool NumericSelectorControlContext::MaxValueReached() const
 {
     return value == maxValue;
 }
+
+bool NumericSelectorControlContext::PaintButtons(Renderer& renderer)
+{
+    ColorPair colorMinus{};
+    ColorPair colorPlus{};
+    if (instance->IsEnabled() == false)
+    {
+        colorMinus = Cfg->Button.Text.Inactive;
+        colorPlus  = Cfg->Button.Text.Inactive;
+    }
+    else if (Focused)
+    {
+        colorMinus = Cfg->Button.Text.Focused;
+        colorPlus  = Cfg->Button.Text.Focused;
+    }
+    else
+    {
+        colorMinus = Cfg->Button.Text.Normal;
+        colorPlus  = Cfg->Button.Text.Normal;
+    }
+
+    if (MinValueReached() && intoInsertionMode == false)
+    {
+        colorMinus = Cfg->Button.Text.Inactive;
+    }
+
+    if (MaxValueReached() && intoInsertionMode == false)
+    {
+        colorPlus = Cfg->Button.Text.Inactive;
+    }
```

Comment:
Sunt niste API-uri facute direct in ControlContext care fac cam acelasi lucru: `GetState` sau `GetContainerState`

Reply:
le-am folosit aici `bool NumericSelectorControlContext::GetTextRenderColor(Graphics::ColorPair& color) const` dar mi-au scapat si aici
will fix